usagov/usagov-2021
# Usagov 281 mobile sticky navigation

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-281

## Description

 Make the header sticky for devices with a screen size less than or equal to 1024px. The mobile sticky nav should work the same way it did in the old version of the website. You can see it here: https://web.archive.org/web/20220311030210/https://www.usa.gov/

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Go to any page in localhost and check that the mobile navbar works the same way it did in the old version of the website. You can see it here: https://web.archive.org/web/20220311030210/https://www.usa.gov/.
2. Change the size of the screen and check if the header is sticky only for devices with a screen size less than or equal to 1024px.
3. Test the previous steps in other browsers (Safari, Google Chrome, Firefox, etc).

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
